Modules/Products/Tests/Feature/ProductCategoryFilterTest.php

```
@@ -0,0 +1,109 @@
+            ->assertCanSeeTableRecords([$pieceProduct, $hourProduct])
+            ->filterTable('unit_id', $piece->id)
+            ->assertCanSeeTableRecords([$pieceProduct])
+            ->assertCanNotSeeTableRecords([$hourProduct]);
+    }
+
+    #[Test]
+    #[Group('multi-tenancy')]
+    public function it_only_offers_categories_of_the_current_tenant_as_filter_options(): void
+    {
+        /* Arrange */
+        $ownCategory = ProductCategory::factory()->for($this->company)->create([
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$ownCategory'. (undefined)

(UnusedLocalVariable)